### ISSUE TITLE
Option to keep new line between selectors in SASS

### ISSUE THREAD

**Issue by MiincGu** [Glavin001/atom-beautify#180]
Hi,

I just want to know if there is an option to keep new line between selectors in sass files.
Like below:

```
.a {
  color: #fff;

  .b {
    color: #999;
  }
}
```

not

```
.a {
  color: #fff;
  .b {
    color: #999;
  }
}
```

**Comment by Glavin001** [comment]
SASS is handled by [PrettyDiff](https://github.com/austincheney/prettydiff).
See https://github.com/Glavin001/atom-beautify/blob/master/lib/language-options.coffee#L424-L426 and https://github.com/Glavin001/atom-beautify/blob/master/lib/langs/less-beautify.coffee

I recommend forwarding this issue over to https://github.com/austincheney/prettydiff and /ccing me or letting me know if there is any progress and if I should update Atom Beautify to support a newer Prettydiff version. Thanks!

**Comment by MiincGu** [comment]
The author of PrettyDiff has added the support of this enhancement minutes ago, and he said he will launch it tomorrow. 

May be you can look forward of that.

**Comment by quicoto** [comment]
Here's the change for "preserve empty lines for CSS like languages" from PrettyDiff from 4 days ago: https://github.com/austincheney/prettydiff/commit/843185f0bf6e182ea6cadd12e4d9c41a95d6e2cf

Is it possible to update beauify with it?

**Comment by MiincGu** [comment]
I don't think this issue has been fixed. 

When I have code like:
```
.a {
  color: #fff;

  .b {
    color: #999;
  }
}
```

I still get 
```
.a {
  color: #fff;
  .b {
    color: #999;
  }
}
```

atom beautifer version: 0.22.3

**Comment by Glavin001** [comment]
Can you confirm that Prettydiff's website does work and that this is an Atom Beautify issue?

**Comment by MiincGu** [comment]
I can confirm that. Anything work perfectly on Prettydiff's website.

**Comment by Glavin001** [comment]
I have updated Prettydiff to 0.11.0 with Atom Beautify v0.22.2 ( see commit history https://github.com/Glavin001/atom-beautify/commits/master ).

If updating to v0.22.3 (latest, next release after v0.22.2) still does not work, then I would ask @Prettydiff if those changes are supported on the Node.js version of Prettydiff and not just the web browser version.

**Comment by prettydiff** [comment]
I will make this feature available for v1.11.2.

https://github.com/prettydiff/prettydiff/issues/61

**Comment by Glavin001** [comment]
Migrating this issue over to #229. I am adding support in Atom Beautify for passing the `preserve` option to Prettydiff.

**Comment by Glavin001** [comment]
Published to v0.22.5

Atom Beautify uses a boolean (true or false) value for `preserve_newlines` option and will translate to `all` in Prettydiff. Support is only for LESS/SCSS/SASS (Prettydiff supported languages).

Thank you for your patients.
